charts/cray-product-catalog/values.yaml

```diff
@@ -25,3 +25,11 @@ kubectl:
   image:
     repository: artifactory.algol60.net/csm-docker/stable/docker-kubectl
     tag: 1.19.15
+migration:
+  image:
+    repository: artifactory.algol60.net/csm-docker/unstable/$APP_NAME 
```

Comment:
Should this not be stable instead of unstable?

Reply:
Helm charts and docker images after the CI are getting generated in Unstable folders not sure why